src/reaction.rs

```diff
@@ -129,6 +129,30 @@ impl fmt::Display for Reactions {
                     .or_insert(1);
             }
         }
+        emoji_frequencies
+    }
+
+    /// Returns a vector of emojis
+    /// sorted in descending order of frequencies.
+    ///
+    /// This function can be used to defined the order
+    /// of reaction display in the UIs.
```

Comment:
there are some typos, it seems, maybe change to something like:

```suggestion
    /// This function can be used to display the reactions sorted in
    ///  the message bubble in the UIs.
```